_woc/Weekly_blogs.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Text Sentiment Analysis Starter Blog"
+excerpt: "This is the inital blog post to derive a generalized interest for the project of Text Sentiment Analysis."
+toc: true
+toc_sticky: true
+toc_label: "Contents"
+
+categories:
+  - woc
+
+tags:
+  - Winter of Code
+  - Developers
+  - Developer Student Clubs
+  - DSC NSEC
+  - DSC IEM
+
+
+last_modified_at: 2020-12-20T08:06:00-05:00
+og_image: /assets/images/WoC2020logo.jpg
+header:
+  teaser: "/assets/images/WoC2020logo.jpg"
+  overlay_image: /assets/images/WoC2020logo.jpg
+  overlay_filter: 0.5 # same as adding an opacity of 0.5 to a black background
+  caption: "Image credit: [**Winter of Code**](https://winterofcode.com/) on [**Instagram**](https://www.instagram.com/winterofcode/)"
+  actions:
+    - label: "View Repository on Github"
+      url: "https://github.com/khanfarhan10/TextSentimentAnalysis"
+author: Dinesh_Kumar
+use_math: true
+---
+## Introduction 
+Blah Blah Blah. 
+My name is **Dinesh Kumar**.I’m an inquisitive thinker and a Python Developer who has his commitments
+towards data science. I am dedicated and always eager to learn more and
+more about data science, machine learning and artificial intelligence
+related knowledge.
+### About You
+I'm **Dinesh Kumar** Currently pursuing my btech from **NIET** greater noida in **Computer Science** department.My Hobbies are to play Cricket and love to listen songs.
+#### Education
+- I have completed my High school Qualification from **Ram-eesh International School** in 2018.
+- And currently I am pursuing my Btech From **Noida Institute of Engineering and Technology** in greater noida from Computer Science Branch.
+- Currently i am in **3rd year** and passout by 2022.
+#### Projects
+- I have prepared many models like **Supervised models**, **Unsupervised model**, **Decision tree model**.
+- I have also done **EDA** on a mall details.
+- In october month i also completed my **Internship** in **Spark's Foundation** as **Data Science and Buisness Analytics**.
```

Comment:
Perhaps provide links??
Add links like this : 
```[My Awesome Website](https://khanfarhan10.github.io/)```
Rendered as :
[My Awesome Website](https://khanfarhan10.github.io/)

Reply:
Or checkout : https://medium.com/@itsjzt/beginner-guide-to-markdown-229adce30074